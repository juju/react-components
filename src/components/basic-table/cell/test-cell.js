/* Copyright (C) 2017 Canonical Ltd. */

import React from 'react';
import enzyme from 'enzyme';

import BasicTableCell from './cell';

describe('BasicTableCell', () => {
  const renderComponent = (options = {}) =>
    enzyme.shallow(
      <BasicTableCell
        classes={options.classes}
        columnSize={options.columnSize || 5}
        content={options.content}
        isLastCol={options.isLastCol}
      />
    );

  it('can render', () => {
    const wrapper = renderComponent({
      content: <span>Content!</span>
    });
    expect(wrapper).toMatchSnapshot();
  });
});
