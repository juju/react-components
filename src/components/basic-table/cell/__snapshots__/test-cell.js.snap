// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BasicTableCell can render 1`] = `
<td
  className="basic-table__cell"
>
  <span>
    Content!
  </span>
</td>
`;
